api/txspool.go

```diff
@@ -40,61 +39,94 @@ func (a *API) postPoolTx(c *gin.Context) {
 	c.JSON(http.StatusOK, receivedTx.TxID.String())
 }
 
+// AtomicGroup represents a set of atomic transactions
+type AtomicGroup struct {
+	AtomicGroupID common.AtomicGroupID `json:"atomicGroupId"`
```

Comment:
Isn’t it better to name `AtomicGroupID` as `ID`, bcs from the code it's understandable, that this ID belongs to AtomicGroup?

Reply:
Applied the change request